parts/transistor/nmos/base-sot-523.json

```diff
@@ -9,7 +9,7 @@
     ],
     "description": [
         false,
-        ""
+        "base N-channel MOSFET"
```

Comment:
I agree it's not a good description. I add it just to avoid "bot: fail". It really nonsense description. I don't know what else to say about BASE part. It would be nice if description would not be necessary for base parts.

Reply:
I think this should include the package, if it must be set. This pase part is package-specific.